test/boost/cql_query_test.cc

```diff
@@ -1121,24 +1121,20 @@ SEASTAR_TEST_CASE(test_range_deletion_scenarios_with_compact_storage) {
 
 SEASTAR_TEST_CASE(test_invalid_range_deletion) {
     cql_test_config cfg;
-    cfg.disabled_features.insert(sstring(gms::features::UNBOUNDED_RANGE_TOMBSTONES));
     return do_with_cql_env_thread([] (cql_test_env& e) {
```

Comment:
You can just remove the test. Just check that asymmetric intervals are tested elsewhere.

Reply:
Removed.